docs/contributors/contributing/e2e-guidelines.md

```diff
@@ -0,0 +1,55 @@
+# E2E Guidelines <!-- omit in toc -->
+
+## Table of contents <!-- omit in toc -->
+
+-   [Structure](#structure)
+-   [Playwright](#playwright)
+    -   [Structure](#structure-1)
+
+This living document serves to prescribe coding guidelines specific to the WooCommerce Blocks project E2E tests. For more information on how to run Playwright end-to-end (E2E) tests, please refer to the [dedicated resource](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/tests/e2e-pw/README.md).
+
+## Structure
+
+There are two folders dedidcate to E2E tests.
+
+The first folder is named "e2e" and it contains all the E2E tests that were created with the deprecated infrastructure Jest + Puppetter. The "e2e-pw" folder contains all the E2E tests that were created with the current infrastructure: Playwright. These tests are actively maintained and should be used for all new E2E testing.
+
+### Playwright
+
+#### Structure
+
+There are two Playwright projects configuration:
+
+- blockTheme
+- classicTheme
+
+The blockTheme project runs the tests with the suffix *block_theme*. In this case, the theme is a block theme. The block theme is the default WordPress theme. Currently, it is Twenty Twenty Three. You should use this configuration if you want test the block with the Site Editor.
```

Comment:
Nit: the [official name](https://wordpress.org/themes/twentytwentythree/) uses a dash. I would keep it consistent so it's easy to find when searching.
```suggestion
The blockTheme project runs the tests with the suffix *block_theme*. In this case, the theme is a block theme. The block theme is the default WordPress theme. Currently, it is Twenty Twenty-Three. You should use this configuration if you want test the block with the Site Editor.
```

Reply:
Fixed with 6c946454ebd8e7523d52f78433cd63c09f9b951f